### PR TITLE
Issue 9280: showEquipment on click

### PR DESCRIPTION
**Pull request by EmilyOng**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

#9280

### CODE REVIEW

File: website/client/components/avatar.vue [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <template lang="pug">
+
+head
+
+  script(src='https://unpkg.com/popper.js')
```
Comment (paglias) [inline website/client/components/avatar.vue:5]: why did you add these new scripts? They're already available as npm packages

File: website/client/components/avatar.vue [hunk 2]
```diff
@@ -179,6 +283,15 @@ export default {
       if (!this.$store.state.spellOptions.castingSpell) return;
       this.$root.$emit('castEnd', this.member, 'user', e);
     },
+    
   },
 };
+
+  new Vue({
```
Comment (paglias) [inline website/client/components/avatar.vue:290]: this is creating a new vue instance, why?

File: website/client/components/avatar.vue [hunk 3]
```diff
@@ -16,22 +23,24 @@
...
-      span(:class="getGearClass('back_collar')")
-      span(:class="getGearClass('body')")
+      span(v-tooltip="showEquipments",:class="getGearClass('back_collar')")
+      span(v-tooltip="showEquipments",:class="getGearClass('body')")
       template(v-for="type in ['base', 'bangs', 'mustache', 'beard']")
         span(:class="'hair_' + type + '_' + member.preferences.hair[type] + '_' + member.preferences.hair.color")
-      span(:class="getGearClass('eyewear')")
-      span(:class="getGearClass('head')")
-      span(:class="getGearClass('headAccessory')")
+      span(v-tooltip="showEquipments",:class="getGearClass('eyewear')")
```
Comment (paglias) [inline website/client/components/avatar.vue:36]: if you pull the latest changes from develop, this should work without having to add new scripts

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by librarianmage** [comment]
@EmilyOng Hello! How are you doing with this? It appears that @paglias has some questions about your PR, and it has a merge conflict. Would you be able to pull from develop?

**Pull request closed without merging**